corehq/apps/app_manager/templates/app_manager/form_view_base.html

```diff
@@ -303,7 +303,7 @@
                     {% trans "Case Management" %}
                 </a>
             </li>
-            {%  if allow_usercase and form.form_type == 'module_form' %}
+            {%  if allow_usercase or is_usercase_in_use and form.form_type == 'module_form' %}
```

Comment:
It shows if the domain is on a standard plan (or higher) or user case is already in use (because the domain's subscription has been downgraded after using user case) and the form is a normal form. `or` and `and` have the same operator precedence, so they are evaluated from left to right.